### PR TITLE
Add more coverage for package content search functionality

### LINKED ISSUE #3495: Create coverage for Content -> Packages page

### PR DESCRIPTION
**Pull request by oshtaier**
Closes #3495
Seems as good enough coverage for that area. Of course, we can think about adding something more, but it will be a little bit advanced scenarios
```
nosetests tests/foreman/ui/test_packages.py
.....
----------------------------------------------------------------------
Ran 5 tests in 365.900s

OK
```

### CODE REVIEW

File: tests/foreman/ui/test_packages.py [hunk 1]
```diff
@@ -123,3 +131,68 @@ def test_positive_check_package_details(self):
...
+            basearch='x86_64',
+            org_id=cls.organization.id,
+            product=PRDS['rhel'],
+            repo=REPOS['rhst7']['name'],
+            reposet=REPOSET['rhst7'],
+            releasever=None,
+        )
+        entities.Repository(id=repo_id).sync()
+
+    @tier3
```
Comment (abalakh) [inline tests/foreman/ui/test_packages.py]: Why tier3? These tests are looking very simple to me - just checking corresponding field values.

Comment (oshtaier) [inline tests/foreman/ui/test_packages.py]: because tier ideology has nothing to do with complexity =)
It is System test, because we create subscription, synchronize real repository and then do content search inside repository for specific package
At least three-four application areas are affected, so I made decision like that

Comment (abalakh) [inline tests/foreman/ui/test_packages.py]: git grep for `upload_manifest(` phrase and take a look on tiers - i'd say 60% of tests are marked as tier2, 40% as tier1 and i can't see any test marked as tier3 just because of subscription upload, pretty much the same picture is for repository sync. Tier3 tests are system tests and we aren't testing system here but checking field values and mentioned preconditions never rose tier for other tests. Without preconditions i can't even see a point of why it could be tier3 - for content search functionality verification that search returns proper results is definitely tier1, that's the most basic functionality we can test.

Comment (abalakh) [inline tests/foreman/ui/test_packages.py]: E.g. api content view module contains:
7 tier2 tests [here](https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/api/test_contentview.py#L1046), 4 [here](https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/api/test_contentview.py#L1497) and 4 [here](https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/api/test_contentview.py#L1629).
Test for syncing your specific repo is marked as tier2 too: https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/api/test_repository.py#L913
UI test which exercises different functionality but uses similar rh repo is marked as tier2 too: https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/ui/test_activationkey.py#L964
Basically every rh repo sync test i see is marked as tier2, can't find both tier1 or tier3. Lots of tests which upload manifest are marked as tier1 but with real repo synchronization it's always tier2. No idea where did you get tier3, the only tests rh repo sync marked as tier3 i can see are the ones which work with errata later (and real hosts) or docker registries etc.

Comment (oshtaier) [inline tests/foreman/ui/test_packages.py]: don't have access to my code at that moment, will submit fix tomorrow. Thanks for comment

**Comment by rochacbruno** [comment]
@svtkachenko @abalakh this PR got 2 ACKS/approval can it be merged?

**Comment by oshtaier** [comment]
@rochacbruno Need to move tier3 tag to tier2. Gonna that tomorrow

**Comment by oshtaier** [comment]
Fixed
